src/components/views/room_settings/RoomProfileSettings.js

```diff
@@ -62,13 +62,18 @@ export default class RoomProfileSettings extends React.Component {
         this._avatarUpload = createRef();
     }
 
-    _uploadAvatar = (e) => {
-        e.stopPropagation();
-        e.preventDefault();
-
+    _uploadAvatar = () => {
         this._avatarUpload.current.click();
     };
 
+    _removeAvatar = () => {
+        this.setState({
+            avatarUrl: null,
+            avatarFile: null,
+            enableProfileSave: true,
+        });
```

Comment:
Don't you need a saveProfile call here?

Reply:
Nope, profile is saved when you click the Save button, thus `enableProfileSave`